.circleci/config-2_1.yml

```diff
@@ -277,7 +277,7 @@ executors:
         type: string
         default: medium
     docker:
-      - image: nastra/cassandra-testing-ubuntu1910-java11-w-dependencies:20200406
```

Comment:
should now use `nastra/cassandra-testing-ubuntu1910-java11-w-dependencies:20200603`.